prescription/prescription.go

```diff
@@ -0,0 +1,127 @@
+package prescription
+
+import (
+	"github.com/tidepool-org/platform/data/types/settings/pump"
+)
+
+const (
+	PrescriptionStateDraft     = "draft"
+	PrescriptionStatePending   = "pending"
+	PrescriptionStateSubmitted = "submitted"
+	PrescriptionStateReviewed  = "reviewed"
+	PrescriptionStateExpired   = "expired"
+	PrescriptionStateActive    = "active"
+	PrescriptionStateInactive  = "inactive"
+
+	PrescriptionTrainingInPerson = "inPerson"
+	PrescriptionTrainingInModule = "inModule"
+
+	PrescriptionTherapySettingInitial              = "initial"
+	PrescriptionTherapySettingTransferPumpSettings = "transferPumpSettings"
+	PrescriptionTherapySettingCPT                  = "cpt" // TODO: Certified Personal Trainer?
+
+	PrescriptionLoopModeClosedLoop  = "closedLoop"
+	PrescriptionLoopModeSuspendOnly = "suspendOnly"
+)
+
+
+type Prescription struct {
+	ID                string           `json:"id" bson:"id"`
+	AccessCode        string           `json:"accessCode,omitempty" bson:"accessCode,omitempty"`
+	FirstName         *string          `json:"firstName,omitempty" bson:"firstName,omitempty"`
+	LastName          *string          `json:"lastName,omitempty" bson:"firstName,omitempty"`
+	Birthday          *string          `json:"birthday,omitempty" bson:"birthday,omitempty"`
+	MRN               *string          `json:"mrn,omitempty" bson:"mrn,omitempty"`
+	Email             *string          `json:"email,omitempty" bson:"email,omitempty"`
+	Sex               *string          `json:"sex,omitempty" bson:"sex,omitempty"`
+	Weight            *Weight          `json:"weight,omitempty" bson:"weight,omitempty"`
+	YearOfDiagnosis   *string          `json:"yearOfDiagnosis,omitempty" bson:"yearOfDiagnosis,omitempty"`
+	PhoneNumber       *string          `json:"phoneNumber,omitempty" bson:"phoneNumber,omitempty"`
+	Address           *Address         `json:"address,omitempty" bson:"address,omitempty"`
+	InitialSettings   *InitialSettings `json:"initialSettings,omitempty" bson:"initialSettings,omitempty"`
+	Training          *string          `json:"training,omitempty" bson:"training,omitempty"`
+	TherapySettings   *string          `json:"therapySettings,omitempty" bson:"therapySettings,omitempty"`
+	LoopMode          *string          `json:"loopMode,omitempty" bson:"loopMode,omitempty"`
+	AcknowledgedTerms *bool            `json:"acknowledgedTerms,omitempty" bson:"acknowledgedTerms,omitempty"`
```

Comment:
Redundant with user account / profile? `termsAccepted`

Reply:
This is intended to be used by the prescriber and it was listed as a requirement. I'll rename it to `prescriberTermsAccepted`.